uber/site_sections/groups.py

```diff
@@ -136,6 +136,26 @@ def index(self, session, message='', order='name', show='all'):
             'approved_tables':   sum(g.tables for g in groups if g.status == c.APPROVED)
         }
 
+    def promo_code_groups(self, session, message=''):
+        groups = sorted(session.query(PromoCodeGroup).options(joinedload('buyer')).all(),
+                        key=lambda g: g.name)
```

Comment:
Is there any reason to use ``sorted`` here instead of ``order_by``?  The latter is probably a little faster and more idiomatic.

Reply:
No special reason! I'll fix that.